### DISCUSSION TITLE
pycord error

### DISCUSSION THREAD

**Discussion by satisfyingdeveloper** [Pycord-Development/pycord#735] [Q&A] [answered]
`import discord
from discord import Embed, commands
from settings import invisibilityColor
from random import randint

class CoinflipStartButtons(discord.ui.View):
    def __init__(self, bot, ctx):
        super().__init__(timeout=30)
        self.bot = bot
        self.ctx = ctx

    @discord.ui.button(label='Присоединиться', style=discord.ButtonStyle.gray)
    async def start_coinflip(self, button: discord.ui.Button, interaction: discord.Interaction):
        await interaction.response.edit_message(embed=Embed(description=f'{self.ctx.author.id}, {interaction.user.id}',
                                                            color=invisibilityColor),
                                                view=ConflipChoose(self.bot, self.ctx, interaction))

class ConflipChoose(discord.ui.View):
    def __init__(self, bot, ctx, interaction):
        super().__init__(timeout=30)
        self.bot = bot
        self.ctx = ctx
        self.unicorNumber = None
        self.crowNumber = None
        self.interaction_id = interaction.user.id
        self.author_id = ctx.author.id

    @discord.ui.button(style=discord.ButtonStyle.gray, emoji='<:unicorn_flip:929471803712241685>')
    async def unicorn_flip(self, button: discord.ui.Button, interact: discord.Interaction):
        if interact.user.id == self.interaction_id or self.author_id == interact.user.id:
            self.unicorNumber = randint(0, 100)
            self.crowNumber = randint(0, 100)
            print(self.unicorNumber, 'unicorn', self.crowNumber, 'sova')
            if self.unicorNumber > self.crowNumber:
                embed = Embed()
                embed.set_image(
                    url='https://cdn.discordapp.com/attachments/840149548440485918/925702100963455026/Comp_5.gif')
                await interact.response.edit_message(embed=embed, view=None)

            elif self.unicorNumber < self.crowNumber:
                embed = Embed()
                embed.set_image(url='https://cdn.discordapp.com/attachments/840149548440485918/925702101588398080/Comp_4.gif')
                await interact.response.edit_message(embed=embed, view=None)
                await interact.response.send_message(content='lol')

    @discord.ui.button(style=discord.ButtonStyle.gray, emoji='<:crowflip:929471751879000094>')
    async def crown_flip(self, button: discord.ui.Button, interaction: discord.Interaction):
        pass
`
discord.errors.InteractionResponded: This interaction has already been responded to before

**Comment by plun1331** [comment, marked as answer]
You can't use `Interaction.response` more than once, if you want to send a message after you've responded consider looking into `Interaction.followup`.

**Reply by satisfyingdeveloper** [reply]
can you give example with followup

**Reply by plun1331** [reply]
> can you give example with followup
```py
await interaction.followup.send(...)
```

**Reply by Lulalaby** [reply]
For more question please join https://discord.gg/pycord
